test/functorch/test_control_flow.py

```diff
@@ -476,6 +579,52 @@ def f(x, y):
         graph = make_fx(f, tracing_mode="symbolic")(x, torch.tensor(False))
         self.assertEqual(graph(x, torch.tensor(True)), f(x, torch.tensor(True)))
 
+    @skipIfTorchDynamo("Graph is not captured by backend if test with dynamo")
```

Comment:
This test doesn't check backward graph. We can use the way that's mentioned in previous comment to test the graph.

Reply:
This test still fails, as the graph module does not contain the backward graph. I have realized that the root cause of this is that in the [forward function of CondAutogradOp](https://github.com/pytorch/pytorch/pull/126911/files#diff-4093d24068320f82997aecfaf4260aa495e3b4322e178cb5625f9555e3f8b0f5R359-R363), the `cond_op` is torch.compiled.

Reply:
This test is now working as expected again